crawlers/reddit/source/database/cache.py

```diff
@@ -5,16 +5,16 @@
 class CacheDB:
     '''Methods to work with Redis cache database'''
 
-    def __init__(self, host: str, port: str, password: str = ''):
+    def __init__(self, url: str):
         try:
-            self.redis = redis.StrictRedis(
-                host=host, port=port, password=password)
-            logger.info(f'Connected to cache database on port: {port}')
+            self.redis = redis.StrictRedis.from_url(url, decode_responses=True)
+            logger.info(f'Connected to cache database on url: {url}')
         except Exception as e:
             logger.error(f'Error while connecting to cache database: {e}')
 
-    def save_data(self, key: str, data: str):
-        self.redis.set(key, data)
+    def add_to_set(self, key: str, id: str):
+        self.redis.sadd(key, id)
```

Comment:
Lets use hashmaps, I think, it will be better, cause we want to add `date` to redis data